blog/_posts/2017-09-12-Issue-253.md

```diff
@@ -0,0 +1,134 @@
+- validation for field ```about``` and set the field ```given_by``` as read-only
+- unit model tests for model Feedback
+
+<br>
+
+Our hardest part on our Issue was to fix this test:
+```python
+    def test_list_feedback_works_as_collector(self):    
+        """
+        Collector is allowed to see list of feedback.
+        """
+        self.client.force_login(user=self.collector)
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
+        self.assertEqual(len(response.data), 2)
+```
+This test should return two instances of our test object and check that there are exactly 2 instances in the test database. There is also a difference between a test database and live database. We discussed this many times until we could fully understand :)
+
+<br>
+
+<script id="infogram_0_a09df277-2245-4bca-bec9-922b039710e6" title="Welcome: Your first project" src="https://e.infogram.com/js/dist/embed.js?yjt" type="text/javascript"></script><div style="padding:8px 0;font-family:Arial!important;font-size:13px!important;line-height:15px!important;text-align:center;border-top:1px solid #dadada;margin:0 30px"><a href="https://infogram.com/a09df277-2245-4bca-bec9-922b039710e6" style="color:#989898!important;text-decoration:none!important;" target="_blank">Welcome: Your first project</a><br><a href="https://infogram.com" style="color:#989898!important;text-decoration:none!important;" target="_blank" rel="nofollow">Infogram</a></div>
+
+<br>
+
+#### We Still Want to Do:
+- add HTTP method PATCH to Feedback
+- change field ```weight``` to FloatField
+- add filters to our Feedback + some other limitations
+- refacture
+- writing a Foodsaving-backend beginner guide. We already started [here](https://github.com/mddemarie/Foodsaving-Documentation/blob/master/foodsaving-backend-code-guide.md).
+
+<br>
+
+## Hackathon & Community
+
+We attended HackDays in ImmobillienScount 11.07.-13.07. and we have got very motivated with Stefan: "We can make it!"
+
+Foodsaving founders invited us to their **Hackathon: 14.07.-16.07**. We met here incredibly smart people - Tilmann Becker  (@tiltec), Nick Sellen (@nicksellen) and Janina Abels (@djahnie) who put their time & energy into making the saving a food waste possible.
+
+<br>
+
+![Hackathon in Berlin](/img/blog/2017/hack_1.png) ![Hackathon in Berlin](/img/blog/2017/hack_2.jpeg)   
+_Enjoing saved food and coding together at the Hackathon, Photos by Nick Sellen_
+
+<br>
+
+And we want to thank our great coaches and mentor! Most of them already appeared in our [first blog post](https://github.com/id-gue/summer-of-code/edit/gh-pages/blog/_posts/2017-07-13-print_hello_world.md) – but [Derek Tingle](https://github.com/tingled)  (@tingled) is new and we are very grateful that he spent a lot of time explaining us many related technical topics we needed a help with. You should see how he navigates through the shell – it's amazing!!! :D
```

Comment:
And we want to thank our great coaches and mentor! Most of them already appeared in our [first blog post](https://github.com/id-gue/summer-of-code/edit/gh-pages/blog/_posts/2017-07-13-print_hello_world.md) – but [Derek Tingle](https://github.com/tingled)  (@tingled) is new, and we are very grateful that he has spent a lot of time explaining many related technical topics we needed a help with. You should see how he navigates through the shell – it's amazing!!! :D